_includes/home/connect.md

```diff
@@ -50,7 +50,7 @@ When you build on Firefox, you build with a community of extension developers. A
 
 [Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
 
-[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/#get-involved-in-the-community)
```

Comment:
Hey, this should actually be linked to `/community/#get-involved-in-the-community`. This doesn't link to the wiki; it should link to https://extensionworkshop.com/community/#get-involved-in-the-community.